packages/frontend-2/components/workspace/moveProject/SelectWorkspace.vue

```diff
@@ -0,0 +1,205 @@
+                </template>
+                <template #actions>
+                  <CommonBadge color="secondary" rounded>
+                    {{ formatName(ws.plan?.name) }}
+                  </CommonBadge>
+                </template>
+              </WorkspaceCard>
+            </button>
+          </div>
+        </div>
+      </div>
+      <p v-else class="text-body-xs text-foreground">
+        Looks like you haven't created any workspaces yet. Workspaces help you easily
+        organise and control your digital projects. Create one to move your project
+        into.
+      </p>
+    </template>
+
+    <WorkspacePlanLimitReachedDialog
+      v-model:open="showLimitDialog"
+      subtitle="Upgrade your plan to move project"
+    >
+      <template v-if="limitReachedWorkspace">
+        <p class="text-body-xs text-foreground-2">
+          The workspace
+          <span class="font-bold">{{ limitReachedWorkspace.name }}</span>
+          is on a {{ formatName(limitReachedWorkspace.plan?.name) }} plan with a limit
+          of 1 project and 5 models. Upgrade the workspace to add more projects.
+        </p>
+      </template>
+    </WorkspacePlanLimitReachedDialog>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { graphql } from '~~/lib/common/generated/gql'
+import type {
+  WorkspaceMoveProjectManager_ProjectFragment,
+  WorkspaceMoveProjectManager_WorkspaceFragment,
+  FullPermissionCheckResultFragment
+} from '~~/lib/common/generated/gql/graphql'
+import { useQuery } from '@vue/apollo-composable'
+import { UserAvatarGroup } from '@speckle/ui-components'
+import { workspaceMoveProjectManagerUserQuery } from '~/lib/workspaces/graphql/queries'
+import { formatName } from '~/lib/billing/helpers/plan'
+import { Roles } from '@speckle/shared'
+
+graphql(`
+  fragment WorkspaceMoveProjectSelectWorkspace_User on User {
+    workspaces {
+      items {
+        ...WorkspaceMoveProjectManager_Workspace
+      }
+    }
+    projects(cursor: $cursor, filter: $filter) {
+      items {
+        ...WorkspaceMoveProjectManager_Project
+      }
+      cursor
+      totalCount
+    }
+  }
+`)
+
+const props = defineProps<{
+  project: WorkspaceMoveProjectManager_ProjectFragment
+  canMoveToWorkspace: (permission: FullPermissionCheckResultFragment) => boolean
+  isLimitReached: (permission: FullPermissionCheckResultFragment) => boolean
+  isSsoRequired: (permission: FullPermissionCheckResultFragment) => boolean
+  getDisabledTooltip: (
+    permission: FullPermissionCheckResultFragment
+  ) => string | undefined
+}>()
+
+const emit = defineEmits<{
+  (
+    e: 'workspace-selected',
+    workspace: WorkspaceMoveProjectManager_WorkspaceFragment
+  ): void
+}>()
+
+const { result, loading: initialLoading } = useQuery(
+  workspaceMoveProjectManagerUserQuery,
+  () => ({
+    cursor: null,
+    filter: {},
+    projectId: props.project.id
+  })
+)
+
+const workspaces = computed(() => result.value?.activeUser?.workspaces.items ?? [])
```

Comment:
I do a similar thing in the navigation, it's fine for now but let's see if we can combine later :)